test/fixtures/grouped/treeAdditionalKeys.js

```diff
@@ -0,0 +1,66 @@
+const data = {
```

Comment:
Should rename these files to match the option name

Reply:
Yes! For the same above reason :(